lib/fastlane/plugin/appicon/actions/appicon_action.rb

```diff
@@ -42,6 +42,10 @@ def self.needed_icons
           :macos => {
             '1x' => ['16x16', '32x32', '128x128', '256x256', '512x512'],
             '2x' => ['16x16', '32x32', '128x128', '256x256', '512x512'],
+          },
+          :tvos => {
+            '1x' => ['400x240'],
+            '2x' => ['800x480']
```

Comment:
The 2x here will automatically double the size so this needs to be dropped down 👇 

```suggestion
            '2x' => ['400x240']
```